code/game/objects/effects/aliens.dm

```diff
@@ -168,7 +170,10 @@
 		return
 	if(loc != acid_t.loc && !isturf(acid_t))
 		loc = acid_t.loc
-	ticks += delta_time * acid_strength
+	if(istype(acid_t, /obj/item/flashlight/flare))
+		ticks += delta_time * min((acid_strength * flare_strength_multiplier), flare_strength_max_melt_rate) // we dont wanna make it possible to melt flares too fast
```

Comment:
Having the typecheck here is snowflake. There should be a proc on the target side to return an acid time or whatever.